plugins/org.jboss.tools.openshift.ui/plugin.xml

```diff
@@ -1280,6 +1316,10 @@
            optional="true">
      </commandParameter>
   </command>
+  		<command
+          id="org.jboss.tools.openshift.ui.command.deployment.envvar"
+          name="Manage Environment Variables">
```

Comment:
the label should be "Manage Environment Variables..." as it opens up a dialog. The (identical) label in the context menu of the properties view is also missing those "..."